Core/include/Acts/Seeding/HoughTransformUtils.hpp

```diff
@@ -253,12 +285,18 @@ class HoughPlane {
 
   YieldType m_maxHits = 0.0f;    // track the maximum number of hits seen
   YieldType m_maxLayers = 0.0f;  // track the maximum number of layers seen
+
   std::pair<std::size_t, std::size_t> m_maxLocHits = {
       0, 0};  // track the location of the maximum in hits
   std::pair<std::size_t, std::size_t> m_maxLocLayers = {
       0, 0};  // track the location of the maximum in layers
-  std::set<std::pair<std::size_t, std::size_t>> m_touchedBins =
-      {};                  // track the bins with non-trivial content
+  std::size_t m_assignBatch{20};
+
+  std::unordered_set<std::size_t>
+      m_touchedBins{};  // track the bins with non-trivial content
```

Comment:
similar as for the other file and this applies for a couple of lines above and below
- `///` for doc
- comment above for more sane line breaks